apps/examples/err_report_demo/err_report_main.c

```diff
@@ -84,7 +84,7 @@ uint8_t g_err_nsuccess;
  * WiFi Manager callback prototypes
  */
 static void prv_sta_connected(wifi_manager_result_e);
-static void prv_sta_disconnected(void);
+static void prv_sta_disconnected(wifi_manager_disconnect_e);
```

Comment:
You should fill the argument name. That's our coding rules.

Reply:
Modified as suggested. Thank you.